planty/users/views.py

```diff
@@ -16,3 +18,15 @@ def register(request):
     else:
         form = UserRegisterForm()
     return render(request, 'users/register.html', {'form': form})
+
+
+@login_required
```

Comment:
Nice one! Do we need to add `@login_required` to any other existing views? (I forgot about this 😅)

Reply:
Other views are supposed to be available for all users, so they don't need @login_required decorator :)